.github/workflows/ci.yml

```diff
@@ -18,15 +18,15 @@ jobs:
         # Python version, because typing sometimes changed between bugfix releases.
         # For available versions, see:
         # https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json
-        python-version: ["3.7", "3.7.1", "3.8", "3.8.0", "3.9", "3.9.0", "3.10", "3.10.0", "3.11-dev"]
+        python-version: ["3.7", "3.7.1", "3.8", "3.8.0", "3.9", "3.9.0", "3.10", "3.10.0", "3.11-dev", "pypy3.9"]
```

Comment:
You may want to also test `pypy3.8` and (perhaps) `pypy3.7`, as all three branches are still being maintained.

Reply:
Let's skip the other pypy versions for now to save on those precious CI minutes. (The Python org ran into troubles before IIRC.)